packages/perseus/src/widgets/radio/__stories__/choice-none-above.stories.jsx

```diff
@@ -17,10 +14,27 @@ type ChoiceNoneAboveStory = {|
 |};
 
 export default ({
-    title: "Perseus/Widgets/Radio/None of the Above",
-    args: {className: "", showContent: false, content: ""},
+    title: "Perseus/Widgets/Radio/Choice None of the Above",
+    args: {showContent: false, content: "This is a possible choice"},
 }: ChoiceNoneAboveStory);
 
+const ChoiceDefaults = {
+    checked: false,
+    rationale: "This is a good rational",
+    correct: true,
+    disabled: false,
+    pos: 0,
+    reviewMode: false,
+    showRationale: false,
+    showCorrectness: false,
+    multipleSelect: false,
+    crossedOut: false,
+    previouslyAnswered: false,
+    apiOptions: {},
+    onChange: () => {},
```

Comment:
**suggest:** In some of the other stories we use the `actions` from `@storybook/addon-actions` to log when callbacks like this are called.  Can you do the same for `onChange`?